core/src/main/scala/besom/internal/Context.scala

```diff
@@ -67,6 +67,13 @@ trait Context extends TaskTracker:
     args: A,
     opts: InvokeOptions
   )(using Context): Output[R]
+
+  private[besom] def call[A: ArgsEncoder, R: Decoder, T <: Resource](
+    token: FunctionToken,
+    args: A,
+    resource: T
+  )(using Context): Output[R]
```

Comment:
not exactly correct, missing CallOptions

Reply:
but we can disregard that for now

Reply:
yes, you are correct, it's a stub only